src/components/elements/inputs/masked-input.js

```diff
@@ -4,7 +4,7 @@ import MaskedInput from 'react-text-mask'
 import createNumberMask from 'text-mask-addons/dist/createNumberMask.js'
 import emailMask from 'text-mask-addons/dist/emailMask.js'
 
-export default class MaskedInputInput extends Component {
+export default class MaskedInputField extends Component {
```

Comment:
This is unrelated to this PR, right? Do we still want to rename it? Just checking.